project_event/models/project_task.py

```diff
@@ -189,12 +194,36 @@ def get_equipment_ids_inside(self):
         room_id = self.env['resource.calendar.room'].browse(self.room_id).id
         return room_id.instruments_ids.ids
 
+    @api.multi
+    def cancel_resources_reservation(self):
+        reservation_event = self.env['calendar.event'].browse(
```

Comment:
@eilst if no reservation_event?

Reply:
It is a cancel method, so for sure there most be one event. Should we add the exception anyway?